addon/components/cf-field-value.js

```diff
@@ -1,34 +1,63 @@
 import Component from "@ember/component";
 import layout from "../templates/components/cf-field-value";
 import { computed } from "@ember/object";
+import { inject as service } from "@ember/service";
+import getFileAnswerInfoQuery from "ember-caluma/gql/queries/get-fileanswer-info";
 
 export default Component.extend({
   layout,
 
+  apollo: service(),
+
   tagName: "span",
 
-  renderValue: computed("field.answer.value", function() {
+  value: computed("field.answer.value", function() {
     const field = this.get("field");
 
     switch (field.question.__typename) {
       case "ChoiceQuestion": {
         const option = field.question.choiceOptions.edges.find(
           edge => edge.node.slug === field.answer.value
         );
-        return option ? option.node.label : field.answer.value;
+        return { label: option ? option.node.label : field.answer.value };
       }
       case "MultipleChoiceQuestion": {
         const answerValue = field.answer.value || [];
         const options = field.question.multipleChoiceOptions.edges.filter(
           edge => answerValue.includes(edge.node.slug)
         );
-        return options && options.length
-          ? options.map(edge => edge.node.label).join(", ")
-          : answerValue.join(", ");
+        return {
+          label:
+            options && options.length
```

Comment:
Is it possible that there are no options but a non-empty answerValue?

Reply:
Not sure. I didn't change anything here, this was just prettier doing some reformatting. Let's leave it for the moment :wink:

Reply:
Okey :-)

Plus. I just realized that the chosen options could have been removed from the question.

Reply:
:+1: Is this good to go, then?